sentinelhub/api/utils.py

```diff
@@ -1,38 +1,44 @@
 """
 Module implementing some common utility functions
 """
+from enum import Enum
+from typing import Dict, Optional, Type, TypeVar
+
 from dataclasses_json import LetterCase
 from dataclasses_json import config as dataclass_config
 
 from ..geometry import Geometry
 from ..time_utils import parse_time, serialize_time
 
+_KT = TypeVar("_KT")
+_VT = TypeVar("_VT")
```

Comment:
I find it more understandable if we have:

```Python
_KeyType = TypeVar("_KeyType")
_ValueType = TypeVar("_ValueType")
```
But I know that pylint is complaining about such notation. PEP8 says only [this](https://peps.python.org/pep-0008/#type-variable-names) about `TypeVar` names. My main concern with names that don't start with underscore is that they would be parsed into documentation. :thinking: 

Update: I checked and Sphinx doesn't show types in docs even if they don't start with underscore. So perhaps we could even write all types without underscore:

```Python
KeyType = TypeVar("KeyType")
ValueType = TypeVar("ValueType")
```
Although this could signify that their scope is larger than just this module. I'm not sure...

Reply:
I'm using `KT` and `VT` since the `Dict` type does it. But they are using it without underscores, if sphinx doesnt show them then we can drop them.

Reply:
Then let's just leave it as it is. If it's clear for Python core developers it should be clear for us too. :sweat_smile: